### DISCUSSION TITLE
What's new with 12.230924 ?

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#306] [General]
Spotted latest SpiralLinux on Distrowatch since yesterday, but changelogs are "invisible" thus far.

**Comment by geckolinux** [comment]
Interesting, they don't normally list stuff on Distrowatch until I notify them. That's just a development folder, but it's close to being finalized.

**Comment by dacomboman** [comment]
Ya, they're quick on the gun at times.

Anyhow, it's good to see SpiralLinux still on board! 👍 
Almost a year day for day since last seen right?

**Reply by geckolinux** [reply]
Yep, the idea with SpiralLinux is for it to be as low-maintenance as possible, both for the users and for the developer. The initial SpiralLinux release came out toward the tail end of the Debian 12 development cycle, so hopefully this next SpiralLinux release will require very few updates before Debian 13 comes out.

**Reply by fweller** [reply]
@geckolinux  I am imagining that a release requires a lot of time to build and test for basic functionality, and that's only for a single desktop flavor.  I am amazed that you are tackling all of the other flavors as well.

**Reply by geckolinux** [reply]
@fweller Mainly testing and solving little issues is where the time goes. This time around there have been quite a few little paper cuts in Debian 12 as well as a major bug to work around in the Debian `live-build` tool that I use to generate the ISOs, but we're almost there.

**Comment by fweller** [comment]
@geckolinux 
I've been testing 12.230924 KDE Plasma edition.  
So far all is good.  One thing I noticed is during install, The Calamares installer thought I was in New York instead of Los Angeles.  I recall this was an issue with last year's released version as well.  Not a big deal really.

**Reply by geckolinux** [reply]
@fweller Nice, thanks for testing! I haven't enabled the geolocation for the installer, so it always defaults to New York.
